command/genesis/polybft_params.go

```diff
@@ -272,6 +272,11 @@ func (p *genesisParams) deployContracts(totalStake *big.Int) (map[types.Address]
 			artifact: contractsapi.L2StateSender,
 			address:  contracts.L2StateSenderContract,
 		},
+		{
+			// EIP1559Burn contract
+			artifact: contractsapi.EIP1559Burn,
```

Comment:
From what I understand, this is a requirement for EIP-1559. But, what would happen if a network starts without it and then enables it? It would break forever I guess.

Reply:
So it would make more sense to don't pre-deploy it here at all. Also, this address could be changed as well. Good point.